### PR TITLE
37 story

### PR DESCRIPTION
**Pull request by chanson02**
@MLHumphries @EngineeringDude123 @NoahBartell Please add your dialogues to the json files.

### CODE REVIEW

File: dunwoody_disaster/dialogues/Joe Axberg.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Cooper": {
+    "protagonist_lines": [
+      "I can already convert hex to decimal! How about a showdown Professor Axberg?",
+      "Bring it on! My virtualization layer can run anywhere!"
+    ],
+    "antagonist_lines": [
+      "*chuckles*, Let's see ify our code can compile to my Operating System!",
```
Comment (chanson02) [inline dunwoody_disaster/dialogues/Joe Axberg.json]:
```suggestion
      "*chuckles*, Let's see if your code can compile to my Operating System!",
```

File: main.py [hunk 2]
```diff
@@ -32,13 +33,47 @@ def __init__(self):
...
+
+    def showSelector(self):
+        self.stack.setCurrentWidget(self.selector)
+
+    def userSelectedCharacter(self, character: Character):
+        self.player = character
+        self.saveCharacter(character)
+        self.mapScreen = MapScreen(Map.buildMap(self.player))
+        self.mapScreen.setStyleSheet("background-color: #41A392;")
+        # self.mapScreen.onEnter(self.EnterFight)
```
Comment (chanson02) [inline main.py]:
```suggestion
```

File: dunwoody_disaster/dialogues/Joe Axberg.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
```
Comment (MLHumphries) [inline dunwoody_disaster/dialogues/Joe Axberg.json]: grammar mistake fyi

**Review by MLHumphries — APPROVED** [review]

File: dunwoody_disaster/dialogues/Joe Axberg.json (hunk 3 above)
Comment (chanson02) [reply dunwoody_disaster/dialogues/Joe Axberg.json]: I made a suggestion for this but now it's outdated lol

**Review by NoahBartell — APPROVED** [review]
Lgtm

**Pull request merged**